.changeset/perfect-mangos-greet.md

```diff
@@ -0,0 +1,24 @@
+---
+"@comet/admin-color-picker": major
+"@comet/admin-react-select": major
+"@comet/admin-date-time": major
+"@comet/blocks-admin": major
+"@comet/admin-icons": major
+"@comet/admin-theme": major
+"@comet/admin-rte": major
+"@comet/cms-admin": major
+"@comet/admin": major
+---
+
+Migrate to MUI v6
```

Comment:
Could you please start the upgrade guide as well?

Reply:
Do you mean this in the docs? 
https://docs.comet-dxp.com/docs/migration/migration-from-v6-to-v7

Reply:
Do I need to add something in https://github.com/vivid-planet/comet-upgrade as well then?

Reply:
> Do you mean this in the docs? (just for v7-to-v8)

Yes

> Do I need to add something in https://github.com/vivid-planet/comet-upgrade as well then?

Theoretically we could execute the MUI codemods in our upgrade script

Reply:
Added to the docs in b9bc8bc05fbeea91305f8951dcb87167a8a0b695
And codemods are executed in https://github.com/vivid-planet/comet-upgrade/pull/23